modules/RTC/TraceablePeerConnection.js

```diff
@@ -896,7 +897,16 @@ TraceablePeerConnection.prototype._remoteTrackAdded = function(stream, track, tr
         return;
     }
 
-    logger.info(`${this} creating remote track[endpoint=${ownerEndpointId},ssrc=${trackSsrc},type=${mediaType}]`);
+    let sourceName = this.signalingLayer.getTrackSourceName(trackSsrc);
```

Comment:
lets execute the logic only if feature flag is enabled and assign `undefined` otherwise

Reply:
the goal is really to not execute any of the new logic when the feature flag is off

For the modified log message below I think it's fine if it will say "undefined" without the feature flag.

Reply:
I don't think lint is allowing me to have it as undefined?

Reply:
lets add ignore exception